code/modules/antagonists/cult/cult.dm

```diff
@@ -470,7 +472,16 @@
 
 	if(members.len)
 		parts += "<span class='header'>The cultists were:</span>"
-		parts += printplayerlist(members)
+		var/list/noharvesterscult = list()
+		for(var/datum/mind/cultmind as anything in members)
+			if(istype(cultmind.current, /mob/living/simple_animal/hostile/construct/harvester))
+				continue
+			if(cultmind in truecult)
+				continue
+			noharvesterscult += cultmind
```

Comment:
I'm not sure what this loop is supposed to catch? Why not just use truecult directly?

Reply:
Just before Narsie summon time we save all cultists to true_cult
then at roundend report print, we skip all harvesters and skip all in true_cult; so we add every person who was converted legit (conversion rune, soulstone)
and then we add those in true_cult, since they might be harvesters cuz of Narsie, but also they might have been actual legit cultists from roundstart so we're interested in them

Reply:
Why not just directly add those in true_cult to the round end report? Wouldn’t that already contain every roundstart cultist, convert, shade and construct without those that were converted afterwards (which you can assume to be only harvesters)?

Reply:
@Time-Green Could do that, but I thought if a Death Squad or ERT or whatever was converted after summon people might want to see that
Not a huge deal of course since the shuttle comes in 2 or 3 mins anyway
Should I just go with true_cult directly?

Reply:
Just go with it directly, the edge case is beyonf extreme, and even then they didn’t contribte to summoning Nar’Sie